test_src/sljitTest.c

```diff
@@ -11339,11 +11376,6 @@ static void test90(void)
 	/* ibuf[4] */
 	ibuf[4] = -824;
 
-	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R4, 0, SLJIT_IMM, WCONST(0xffff8192a3b4, 0x8192a3b4));
-	sljit_emit_op1(compiler, SLJIT_REV_S32, SLJIT_S4, 0, SLJIT_R4, 0);
-	/* buf[6] */
-	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_MEM1(SLJIT_S0), 6 * sizeof(sljit_sw), SLJIT_S4, 0);
-
```

Comment:
maybe this test should had been `IS_64BIT` and focus on the "store" of a truncated 32bit signed instead?

see the case added to test89 for details.